### DISCUSSION TITLE
When I create a new document, the type'/' to insert, or start writing does not show up.

### DISCUSSION THREAD

**Discussion by belliott71** [outline/outline#6934] [Show and tell]
I have outline (0.76.20) installed in a docker container with nginx and reverse proxy setup as a subdomain with keycloak.  Whenever I create a new collection and new document, type'/' to insert, or start writing does not show up.  I can move my mouse around and the cursor will change from pointer to text input icon.  If I hit the right spot, the hand pointing icon will show up.  when I locate the text input icon and start typing the text is invisible.  When I refresh the page, the text will show up and I can delete the text that was entered and the plus icon appears and all is fine after that.  I just have to go through all that workflow of finding the right spot to type and refresh the page to delete the text.  I have seen all the discussions about not being able to edit and have changed my location blocks to match, but nothing matters.  I have chown the host directory to 1001 to make sure it wasnt a permission error.  Has anyone else experienced this behavior and had a fix for it?  Thanks

**Comment by tommoor** [comment]
It sounds like the websocket isn't connecting, which you'd be able to see in the network inspector. In which case I suspect your changes are not persisted at all and it might just be rendering a local cache – take a look in the database.

In terms of the specific issue I don't know but it will be related to the proxy, there are a lot of other similar discussions that were fixed with the correct proxy configuration.

**Reply by belliott71** [reply]
Thans for the reply.  The changes do persist as I can create the document after going through all the loops.  I can create on one pc and it shows up on another.  I did inspect network activity and see these 2 entries
Mixed Content: The page at '<URL>' was loaded over HTTPS, but attempted to connect to the insecure WebSocket endpoint 'ws://outline.mysite.com/collaboration/document.37275cfe-5416-421a-a3b2-e9631635bff7'. This request has been blocked; this endpoint must be available over WSS.
AND
websocket.js:44 WebSocket connection to 'wss://outline.mysite.com/realtime/?EIO=4&transport=websocket' failed: 

i did add a location block for collaboration and realtime endpoints per this post and the error still persists
[https://github.com/chsasank/outline-wiki-docker-compose/issues/35](url)

**Reply by belliott71** [reply]
I found the fix for the mixed content error.  I added this to the server block in nginx.
add_header Content-Security-Policy upgrade-insecure-requests;
Now I can edit after new document created
